.drone.yml

```diff
@@ -433,6 +433,13 @@ steps:
   failure: always
   image: grafana/docker-puppeteer:1.0.0
   name: test-a11y-frontend
+- commands:
```

Comment:
I'm pretty sure this shouldn't have any changes relative to main. Looks like something happened here? 🤔